blocks/url-input/index.js

```diff
@@ -104,6 +104,12 @@ class UrlInput extends Component {
 
 	onKeyDown( event ) {
 		const { selectedSuggestion, posts } = this.state;
+		// If the suggestions are not shown, we shouldn't handle the arrow keys
+		// We shouldn't preventDefault to allow block arrow keys navigation
```

Comment:
Doesn't `return false;` have the same effect as `preventDefault`?

Reply:
I guess not since it fixed the issue (maybe it was the stopPropagation) but any way safer to just return